code/modules/client/preference_setup/loadout/loadout_eyes_vr.dm

```diff
@@ -1,9 +1,12 @@
 /datum/gear/eyes/medical
-	allowed_roles = list("Medical Doctor","Chief Medical Officer","Chemist","Paramedic","Geneticist", "Psychiatrist", "Field Medic")
+	allowed_roles = list("Medical Doctor", "Chief Medical Officer", "Chemist", "Paramedic", "Geneticist", "Psychiatrist", "Field Medic", "Talon Doctor")
 
 /datum/gear/eyes/meson
 	display_name = "Optical Meson Scanners (Engineering, Science)"
-	allowed_roles = list("Station Engineer","Chief Engineer","Atmospheric Technician", "Scientist", "Research Director", "Explorer", "Pathfinder")
+	allowed_roles = list("Station Engineer", "Chief Engineer", "Atmospheric Technician", "Scientist", "Research Director", "Explorer", "Pathfinder", "Talon Engineer")
+
+/datum/gear/eyes/security
+	allowed_roles = list("Security Officer", "Head of Security", "Warden", "Detective", "Talon Guard")
```

Comment:
tbh Talon guard has no use for security HUD since its mostly people for seeing who is set to arrest and only Tether can do that.

Reply:
there's also no way talon would have access to NT's security or medical records, more to the point

perhaps a better solution would be to make some talon-specific eyewears that have some of the overlay features and other perks enabled (flash resist for marine/guard, etc.) but none of the records access?

Reply:
Good call, I think it'd be easy enough to make a child subtype of those HUDs with restricted functionality

Cheers for the suggestions.

Reply:
Note to self: 'legacy'/'surplus'/'retrofitted' HUDs for namesake.